src/App.tsx

```diff
@@ -1,223 +1,142 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React, { useContext, useEffect, useRef } from 'react';
+import React, {
+  useContext, useEffect, useState, useMemo,
+} from 'react';
+import { getTodos, removeTodo } from './api/todos';
 import { AuthContext } from './components/Auth/AuthContext';
+import { ErrorNotification } from './components/ErrorNotification';
+import { Footer } from './components/Footer';
+import { Header } from './components/Header';
+import { TodoList } from './components/TodoList';
+import { Filter } from './types/Filter';
+import { Todo } from './types/Todo';
 
 export const App: React.FC = () => {
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
   const user = useContext(AuthContext);
-  const newTodoField = useRef<HTMLInputElement>(null);
+
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filterOption, setFilterOption] = useState(Filter.All);
+  const [isError, setIsError] = useState(false);
+  const [errorText, setErrorText] = useState('');
+  const [isAdding, setIsAdding] = useState(false);
+  const [newTodo, setNewTodo] = useState<Todo | null>(null);
+  const [isLoading, setIsLoading] = useState(false);
+  const [deletedTodoId, setDeletedTodoId] = useState([0]);
```

Comment:
```suggestion
  const [deletedTodoIds, setDeletedTodoIds] = useState([0]);
```
in the plural, it contains an array of IDs :arrow_up:

Reply:
and it should be an empty array by default, no?